File: 02_Programming_Fundamentals/04_Programming_Fundamentals_PHP/03_Intro_And_Basic_Syntax_More_Exercise/05_messages.php
<?php

$n = readline();
$msg = "";
for ($i = 0; $i < $n; $i++) {
  $input = readline();

  switch ($input) {
      case "0":
          $msg .= " ";
          break;
      case "2":
          $msg .= "a";
          break;
      case "22":
          $msg .= "b";
          break;
      case "222":
          $msg .= "c";
          break;
      case "3":
          $msg .= "d";
          break;
      case "33":
          $msg .= "e";
          break;
      case "333":
          $msg .= "f";
          break;
      case "4":
          $msg .= "g";
          break;
      case "44":
          $msg .= "h";
          break;
      case "444":
          $msg .= "i";
          break;
      case "5":
          $msg .= "j";
          break;
      case "55":
          $msg .= "k";
          break;
      case "555":
          $msg .= "l";
          break;
      case "6":
          $msg .= "m";
          break;
      case "66":
          $msg .= "n";
          break;
      case "666":
          $msg .= "o";
          break;
      case "7":
          $msg .= "p";
          break;
      case "77":
          $msg .= "q";
          break;
      case "777":
          $msg .= "r";
          break;
      case "7777":
          $msg .= "s";
          break;
      case "8":
          $msg .= "t";
          break;
      case "88":
          $msg .= "u";
          break;
      case "888":
          $msg .= "v";
          break;
      case "9":
          $msg .= "w";
          break;
      case "99":
          $msg .= "x";
          break;
      case "999":
          $msg .= "y";
          break;
      case "9999":
          $msg .= "z";
          break;
      default:
          echo "Invalid combination" . PHP_EOL;
  }

}

echo $msg;

?>

File: 02_Programming_Fundamentals/02_Programming_Fundamentals_with_C#/03_Intro_And_Basic_Syntax_More_Exercise/05_messages.cs
using System;

class MainClass {
  public static void Main (string[] args) {
  
  int n = int.Parse(Console.ReadLine());
  string msg = "";

  for (int i = 0; i < n; i++) {
    int input = int.Parse(Console.ReadLine());

    if (input == 0) { msg += " "; }
    else if (input == 2) { msg += "a"; }
    else if (input == 22) { msg += "b"; }
    else if (input == 222) { msg += "c"; }
    else if (input == 3) { msg += "d"; }
    else if (input == 33) { msg += "e"; }
    else if (input == 333) { msg += "f"; }
    else if (input == 4) { msg += "g"; }
    else if (input == 44) { msg += "h"; }
    else if (input == 444) { msg += "i"; }
    else if (input == 5) { msg += "j"; }
    else if (input == 55) { msg += "k"; }
    else if (input == 555) { msg += "l"; }
    else if (input == 6) { msg += "m"; }
    else if (input == 66) { msg += "n"; }
    else if (input == 666) { msg += "o"; }
    else if (input == 7) { msg += "p"; }
    else if (input == 77) { msg += "q"; }
    else if (input == 777) { msg += "r"; }
    else if (input == 7777) { msg += "s"; }
    else if (input == 8) { msg += "t"; }
    else if (input == 88) { msg += "u"; }
    else if (input == 888) { msg += "v"; }
    else if (input == 9) { msg += "w"; }
    else if (input == 99) { msg += "x"; }
    else if (input == 999) { msg += "y"; }
    else if (input == 9999) { msg += "z"; }
  } 

  Console.WriteLine(msg);

  }
}